Dockerfile

```diff
@@ -63,7 +63,7 @@ RUN apt-get -qq update && \
 WORKDIR "/root/gazebo"
 
 # ---PREPARE GZ-RENDERING DIRECTORY---
-RUN git clone -b 7-2023.05.31 --single-branch https://github.com/DeepX-inc/gz-rendering.git && \
+RUN git clone -b update/sync-upstream-7 --single-branch https://github.com/DeepX-inc/gz-rendering.git && \
```

Comment:
Sorry the correct branch should be [gz-rendering7](https://github.com/DeepX-inc/gz-rendering/tree/gz-rendering7).
`update/sync-upstream-7` was for a [PR](https://github.com/DeepX-inc/gz-rendering/pull/12) that was done incorrectly, and closed without merging. I have deleted it.

Reply:
Done! I hope it passes...